cluster/manager_delete.go

```diff
@@ -61,7 +91,7 @@ func (m *Manager) deleteCluster(ctx context.Context, cluster CommonCluster, forc
 
 	if !(force && c == nil) {
 		// delete deployments
-		err = helm.DeleteAllDeployment(c)
```

Comment:
Why is helm deployment deletion completely removed? It could contain created resources which are not being deleted by the new code. (Note that deployment != deployment in this case, also the correct terminology is release in case of helm....which (I know) is a confusing name)

Reply:
True that I'll put it back